### PR TITLE
added railway.app hosting site link

### PR DESCRIPTION
**Pull request by Stroller15**
added railway.app hosting site link

### CODE REVIEW

**Comment by Stroller15** [comment]
@TreciaKS pls review my PR

File: README.md [hunk 1]
```diff
@@ -4,21 +4,23 @@
...
-- [Our Contributors](#our-contributors)
-- [License](#license)
-- [Support](#support)
+  - [Welcome to LinksHub 👋](#welcome-to-linkshub-)
+  - [Demo 💻](#demo-)
+  - [Tech stack 📚](#tech-stack-)
+  - [Socials 📱](#socials-)
+  - [Getting Started 👩‍💻](#getting-started-)
+  - [Let's jump right in🌟](#lets-jump-right-in)
+  - [_Want to add your favorite links into the Hub? make sure to follow CONTRIBUTING guidelines._](#want-to-add-your-favorite-links-into-the-hub-make-sure-to-follow-contributing-guidelines)
```
Comment (CBID2) [inline README.md:13]:
```suggestion
  - [_Want to add your favorite links to the Hub? Follow the CONTRIBUTING guidelines._](#want-to-add-your-favorite-links-into-the-hub-make-sure-to-follow-contributing-guidelines)
```

File: README.md [hunk 2]
```diff
@@ -4,21 +4,23 @@
...
+  - [Our Contributors ✨](#our-contributors-)
+  - [License 📝](#license-)
+  - [Support ⭐](#support-)
 
 <a name="welcome-to-linkshub"></a>
 
 # Welcome to LinksHub 👋
 
-LinksHub is a Hub of Links `by Developers for Developers`. Here, we've gathered a collection of all the best and most useful resources, both free and paid, to aid in the development journey.
+LinksHub is a Hub of Links `For Developers By Developers`. Here, we've gathered a collection of all the best and most useful resources, both free and paid, to aid in the development journey.
```
Comment (CBID2) [inline README.md:23]:
```suggestion
LinksHub is a Hub of Links `For Developers By Developers`. Here, we've gathered a collection of all the best and most useful resources, both free and paid, to aid developers of all levels in their development journey.
```

File: README.md [hunk 3]
```diff
@@ -4,21 +4,23 @@
...
 
 We recognize that there's a wealth of information available, but often, it's a matter of knowing where to find it. That's why we've made it our mission to bring the right resources to the right developers.
```
Comment (CBID2) [inline README.md:25]:
```suggestion
We recognize that there's a wealth of information available. However, what's important is _knowing_ where to find it. That's why we've made it our mission to bring the right resources to the right developers.
```

File: database/resources/e-book.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "name":"The Deep Learning textbook",
+        "description":"The Deep Learning textbook is a resource intended to help students and practitioners enter the field of machine learning in general and deep learning in particular. The online version of the book is now complete and will remain available online for free.",
```
Comment (CBID2) [inline database/resources/e-book.json:4]:
```suggestion
        "description":"The Deep Learning textbook is a resource intended to help students and practitioners enter the field of machine learning in general and deep learning in particular. The online version of the book is now complete and will remain online for free.",

```

File: database/youtube/css.json [hunk 5]
```diff
@@ -22,5 +22,13 @@
         "category": "youtube",
         "subcategory": "css",
         "language": "english"
+    },
+    {
+        "name": "Animation Coding",
+        "description": "A wide range of educational videos covering various topics such as animation, Motion Graphic Design, 3D animation, responsive website design, HTML, CSS, JavaScript, UX/UI design, After Effects, Photoshop, Adobe Illustrator, and much more.",
```
Comment (CBID2) [inline database/youtube/css.json:28]:
```suggestion
        "description": "This site has educational videos that cover various topics such as animation, Motion Graphic Design, 3D animation, responsive website design, HTML, CSS, JavaScript, UX/UI design, After Effects, Photoshop, Adobe Illustrator, and more.",
```

File: database/youtube/machine-learning.json [hunk 6]
```diff
@@ -1,8 +1,34 @@
...
-    "description": "3Blue1Brown, by Grant Sanderson, is some combination of math and entertainment, depending on your disposition. The goal is for explanations to be driven by animations and for difficult problems to be made simple with changes in perspective.",
-    "url": "https://www.youtube.com/@3blue1brown",
-    "category": "youtube",
-    "subcategory": "machine-learning",
-    "language": "english"
-}]
+[
+    {
+        "name":"Udacity",
+        "description":"Free online course for learning tensorflow for Deep learning",
```
Comment (CBID2) [inline database/youtube/machine-learning.json:4]:
```suggestion
        "description":"Free online course for learning TensorFlow for Deep learning",
```

File: database/youtube/machine-learning.json [hunk 7]
```diff
@@ -1,8 +1,34 @@
...
+        "url":"https://learn.udacity.com/courses/ud187",
+        "category":"online platform",
+        "subcategory":"tensorflow",
+        "language":"english"
+    },
+    {
+        "name": "3Blue1Brown",
+        "description": "3Blue1Brown, by Grant Sanderson, is a combination of math and entertainment, depending on your disposition. The goal is for difficult problems to be made simple with changes in perspective.",
```
Comment (CBID2) [inline database/youtube/machine-learning.json:12]:
```suggestion
        "description": "3Blue1Brown, by Grant Sanderson, is a combination of math and entertainment. The goal is for difficult problems to be made simple with changes in perspective.",
```

File: database/youtube/machine-learning.json [hunk 8]
```diff
@@ -1,8 +1,34 @@
...
+        "name":"Google",
+        "description":"Free online course provided by google to learn machine learning and it's implementations",
+        "url":"https://developers.google.com/machine-learning",
+        "category":"online platform",
+        "subcategory":"machine-learning",
+        "language":"english"
+    },
+    {
+        "name":"FreeCodeCamp",
+        "description":"Learn Machine Learning in a way that is accessible to absolute beginners. You will learn the basics of Machine Learning and how to use TensorFlow to implement many different concepts.",
```
Comment (CBID2) [inline database/youtube/machine-learning.json:28]:
```suggestion
        "description": "In this beginner-level course, you will learn the basics of Machine Learning and how to use TensorFlow to implement different techniques.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Stroller15! 😄 You're off to a great start! 😄 I made some suggestions on how to improve your descriptions. Well done! 👍🏾

**Review by rupali-codes — COMMENTED** [review]
@Stroller15 did you use `git add .` after updating your branch? 
 Please only stage the files you have made changes in, not more than that✨

**Comment by Stroller15** [comment]
> @Stroller15 did you use `git add .` after updating your branch? Please only stage the files you have made changes in, not more than that✨

no i added only blogs.json and I thing prblm is dev branch

**Pull request closed without merging**